### DISCUSSION TITLE
ZOAU Hot Topics article covering zinfo

### DISCUSSION THREAD

**Discussion by AnthonyGiorgio** [IBM/zoau-samples#48] [General]
Interested in ZOAU?  Then check out [the latest Hot Topics article](https://zos-hot-topics.com/2022/zinfo-utility/)!  It covers the new `zinfo` utility that you can use for fact gathering on your z/OS system.

**Comment by Degilio** [comment]
Anthony the link leads me to a 404


Frank J. De Gilio
CTO for Mainframe DeClunkification
IBM Systems Group

E-mail: ***@***.******@***.***>
Cell: 845-546-1130



From: Anthony Giorgio ***@***.***>
Date: Friday, October 14, 2022 at 15:39
To: IBM/zoau-samples ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EXTERNAL] [IBM/zoau-samples] ZOAU Hot Topics article covering zinfo (Discussion #48)
Interested in ZOAU? Then check out the latest Hot Topics article! It covers the new zinfo utility that you can use for fact gathering on your z/OS system. https: //zos-hot-topics. com/2022/zinfo-uti — Reply to this email directly, ‍ ‍ ‍ ‍
ZjQcmQRYFpfptBannerStart
This Message Is From an External Sender
This message came from outside your organization.
ZjQcmQRYFpfptBannerEnd

Interested in ZOAU? Then check out the latest Hot Topics article! It covers the new zinfo utility that you can use for fact gathering on your z/OS system.

https://zos-hot-topics.com/2022/zinfo-uti<https://zos-hot-topics.com/2022/zinfo-uti>
